documentation/modules/auxiliary/scanner/discovery/udp_probe.md

```diff
@@ -0,0 +1,70 @@
+## Description
+
+The “udp_probe” module scans a given range of hosts for common UDP services. Note: This module is deprecated and may disappear at any time.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/discovery/ipv6_neighbor```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set THREADS [number of threads]```
+4. Do: ```run```
+
+There are very few required settings for this module so we just configure the RHOSTS and THREADS values and let it run.
+
+## Scenarios
+
+**Running the scanner**
+```
+msf > use auxiliary/scanner/discovery/udp_probe
+
+[!] ******************************************************************************************
+[!] *                 The module scanner/discovery/udp_probe is deprecated!                  *
+[!] *                       It will be removed on or about 2016-11-23                        *
+[!] *                   Use auxiliary/scanner/discovery/udp_sweep instead                    *
+[!] ******************************************************************************************
+msf auxiliary(udp_probe) > show options
```

Comment:
remove show options

Reply:
Rather than document deprecated modules, we should remove them.  

https://github.com/rapid7/metasploit-framework/pull/9317

Reply:
removed.